.github/workflows/test.yml

```diff
@@ -49,6 +73,34 @@ jobs:
       - name: lint
         run: composer lint
 
+      - name: PHP CS Fixer for ALL
+        if: ${{ needs.filter_paths.outputs.fixer_all == 'true' }}
+        id: fixer-all
+        continue-on-error: true
```

Comment:
could you remove `continue-on-error: true` ?

there is no reasons to use this.

Reply:
Thank you for your comment.
I have corrected it in commit [0c05b68](https://github.com/litalico-engineering/eg-r2/pull/23/commits/0c05b68716598626e482f018261c9d330ad94494)